homeassistant/components/poolstation/sensor.py

```diff
@@ -0,0 +1,110 @@
+"""Support for Poolstation sensors."""
+from __future__ import annotations
+
+from pypoolstation import Pool
+
+from homeassistant.components.sensor import SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import DEVICE_CLASS_TEMPERATURE, PERCENTAGE, TEMP_CELSIUS
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import PoolstationDataUpdateCoordinator
+from .const import COORDINATORS, DEVICES, DOMAIN
+from .entity import PoolEntity
+
+PH_SUFFIX = " pH"
+TEMPERATURE_SUFFIX = " Temperature"
+SALT_SUFFIX = " Salt concentration"
+ELECTROLYSIS_SUFFIX = " Electrolysis"
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the poolstation sensors."""
+    stations = hass.data[DOMAIN][config_entry.entry_id][DEVICES]
+    coordinators = hass.data[DOMAIN][config_entry.entry_id][COORDINATORS]
+    entities: list[PoolEntity] = []
+    for pool_id, pool in stations.items():
+        coordinator = coordinators[pool_id]
+        entities.append(PoolPhSensor(pool, coordinator))
+        entities.append(PoolTemperatureSensor(pool, coordinator))
+        entities.append(PoolSaltConcentrationSensor(pool, coordinator))
+        entities.append(PoolElectrolysisSensor(pool, coordinator))
+
+    async_add_entities(entities)
+
+
+class PoolPhSensor(PoolEntity, SensorEntity):
```

Comment:
Instead of creating a new class for each entity, it would be simpler to use `SensorEntityDescription`